pkg/templates/aws/resources/ecs_task_definition.yaml

```diff
@@ -28,6 +28,13 @@ properties:
   EnvironmentVariables:
     type: map(string,string)
     description: A map of the environment variables to pass to the container
+    key_property:
+      sanitize: |
+        {{ .
+          | replace `[^[:alnum:]_]+` "_"
+          | replace `^[^a-zA-Z]+` ""
+          | upper
+        }}
```

Comment:
i dont know that this actually follows the same stuff as lambda since ecs tests were passing, but not blocking